shared/domain/src/test/java/com/ivy/domain/ArbStatSummary.kt

```diff
@@ -0,0 +1,25 @@
+package com.ivy.domain
+
+import com.ivy.data.model.primitive.NonNegativeInt
+import com.ivy.data.model.testing.assetCode
+import com.ivy.data.model.testing.value
+import com.ivy.domain.model.StatSummary
+import io.kotest.property.Arb
+import io.kotest.property.arbitrary.bind
+import io.kotest.property.arbitrary.map
+import io.kotest.property.arbitrary.nonNegativeInt
+
+fun Arb.Companion.statSummary(): Arb<StatSummary> {
+    return Arb.bind(
+        Arb.nonNegativeInt(),
```

Comment:
We don't use this, remove it. Also look how I build Arbs in the project - there was a nicer way